src/network/networkd-dhcp-server-bus.h

```diff
@@ -0,0 +1,9 @@
+/* SPDX-License-Identifier: LGPL-2.1+ */
+#pragma once
+
+#include "sd-bus.h"
+#include "networkd-link.h"
+
+extern const sd_dhcp_server_cb dhcp_server_cb;
+
+extern const sd_bus_vtable dhcp_server_vtable[];
```

Comment:
We gained BusImplemention in the meantime. Please do not expose the raw vtable, but wrap it in BusImplementation.

Reply:
@keszybz what's that? I can't find it with grep or history.

Reply:
See a3b7cf5013e4c3cbe0824b14463e17ef26956f88 for an example. But I see that the path is non-constant here, so it will not work. So this is ok as is.